app/hooks/utils/useDataFetcher.ts

```diff
@@ -116,13 +115,14 @@ export const useFetchArtists = (options: CommonFetcherProps) => {
         return spotifyDataFetcher.fetchArtists(params);
       }
     },
-    {
-      enabled: enabled && !options.lazy,
-      // TODO: Figure out a better way to deal with `after` param
-      getNextPageParam: (lastPage) =>
-        service === "spotify" ? lastPage?.after : lastPage?.nextPageParam,
-    }
-  );
+    enabled: enabled && !options.lazy,
+    // TODO: Figure out a better way to deal with `after` param
+    getNextPageParam: (lastPage) =>
+      service === "spotify"
+        ? (lastPage?.after as any)
+        : lastPage?.nextPageParam,
+    initialPageParam: 0,
```

Comment:
This code is sketchy. Let's definitely check to make sure this still works as expected

Reply:
Looks like it still works 👍 The infinite scrolling is broken though (unrelated to these changes), but that can be resolved in a followup